tests/results/DiffTest.py

```diff
@@ -37,13 +37,15 @@ def test_delete_line(self):
         additions, deletions = self.uut.stats()
         self.assertEqual(deletions, 1)
         self.assertRaises(ValueError, self.uut.delete_line, 0)
+        self.assertRaises(ValueError, self.uut.delete_line, 10)
 
     def test_delete_lines(self):
-        self.uut.delete_lines(1, 10)
-        self.uut.delete_lines(10, 20)
+        self.uut.delete_lines(1, 2)
+        self.uut.delete_lines(2, 3)
         additions, deletions = self.uut.stats()
-        self.assertEqual(deletions, 20)
-        self.assertRaises(ValueError, self.uut.delete_lines, 0, 10)
+        self.assertEqual(deletions, 3)
```

Comment:
Wondering why to change 10,20 to 2,3. Please keep the original values if it doesn't test any case that might have been missed with previous values (10 and 20)